src/Signer/Rsa.php

```diff
@@ -52,7 +55,7 @@
      *
      * @throws InvalidArgumentException
      */
-    private function validateKey($key): void
+    protected function validateKey($key): void
```

Comment:
Please revert this to `private`

Reply:
You can revert https://github.com/lcobucci/jwt/pull/260/commits/235d4870b7977d9b29f3fa9fd756411dc294eaf6 to achieve that